Cargo.toml

```diff
@@ -1,52 +1,52 @@
 [package]
-name = "foundry-fork-db"
+authors = ["Foundry Contributors"]
 description = "Fork database used by Foundry"
-version = "0.1.0"
 edition = "2021"
-rust-version = "1.76"
-authors = ["Foundry Contributors"]
-license = "MIT OR Apache-2.0"
 homepage = "https://github.com/foundry-rs/foundry-fork-db"
+license = "MIT OR Apache-2.0"
+name = "foundry-fork-db"
 repository = "https://github.com/foundry-rs/foundry-fork-db"
+rust-version = "1.76"
+version = "0.1.0"
 
 [lints]
+rust.rust_2018_idioms = {level = "deny", priority = -1}
 rust.unused_must_use = "deny"
-rust.rust_2018_idioms = { level = "deny", priority = -1 }
 rustdoc.all = "warn"
 
 [lints.clippy]
 missing_const_for_fn = "warn"
-use_self = "warn"
 option_if_let_else = "warn"
+use_self = "warn"
 
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-alloy-primitives = "0.7.3"
-alloy-provider = { version = "0.1.2", default-features = false }
-alloy-rpc-types = { version = "0.1.2", features = ["eth"] }
-alloy-serde = { version = "0.1.2", default-features = false }
-alloy-transport = { version = "0.1.2", default-features = false }
+alloy-primitives = "0.7.7"
+alloy-provider = {version = "0.1.4", default-features = false}
+alloy-rpc-types = {version = "0.1.4", features = ["eth"]}
+alloy-serde = {version = "0.1.4", default-features = false}
+alloy-transport = {version = "0.1.4", default-features = false}
```

Comment:
I'm being very pedantic here but please keep the surrounding white space i `{}`

Reply:
Fixed all the white spaces my Code ate :)